core/src/main/javascript/crypto/EccCryptoContext.js

```diff
@@ -93,8 +108,25 @@ var EccCryptoContext;
 
         /** @inheritDoc */
         verify: function verify(data, signature, callback) {
+            var self = this;
             AsyncExecutor(callback, function() {
-                throw new MslCryptoException(MslError.VERIFY_NOT_SUPPORTED, "ECC does not verify");
+                if (!this.publicKey)
+                    throw new MslCryptoException(MslError.VERIFY_NOT_SUPPORTED, "no public key");
```

Comment:
Is MslError.VERIFY_NOT_SUPPORTED the appropriate error code for this failure condition?